### DISCUSSION TITLE
Why isn't Wpf-UI.Tray just built in with the Wpf-UI nugetpackage?

### DISCUSSION THREAD

**Discussion by WafaIda** [lepoco/wpfui#1044] [Q&A] [answered]
I've been wondering for a while why this the Wpf-UI.Tray nuget package isn't just built in with Wpf-UI package.

**Comment by Veritera** [comment]
That's a really good question :D

**Comment by Taiizor** [comment]
I believe this question was asked before. If I remember correctly, it was due to dependencies.

**Comment by pomianowski** [comment, marked as answer]
Tray was separated because:
1. Not everyone has to want to use it
2. To not have any external dependency in the default WPF UI package (System.Drawing.Common in this case)
